partials/tabs/coalaonline.html

```diff
@@ -146,8 +146,8 @@ <h4>Optional Settings</h4>
         <div class="row blue-grey darken-4 white-text z-depth-1 no-margin">
           <span>
             <a class="waves-effect grey-text text-lighten-2">
-              <i ng-click="add_bears(section)" class="fa fa-plus-circle bears-icon-small" aria-hidden="true"> &nbsp;ADD BEAR
-              </i>
+              <i ng-click="add_bears(section)" class="fa fa-plus-circle bears-icon-small" aria-hidden="true"></i>
+              &nbsp;ADD BEAR
```

Comment:
`&nbsp;` not required, the icon has enough padding.

Reply:
I am very sorry I dragged this issue so long because of my small mistakes.

Reply:
Don't worry, this shows how insistent you are on learning.